apple-nav/src/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import { Route, Link } from "react-router-dom";
+import "./App.css";
+import MainNav from "./components/MainNav";
+import SubNav from "./components/SubNav";
+import DummyData from "./data";
+import SubItem from "./components/SubItem";
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: DummyData
+    };
+  }
+  render() {
+    return (
+      <div className="App">
+        <Route
+          path="/"
+          render={props => <MainNav {...props} data={this.state.data} />}
+        />
+        <Route
+          path="/:itemName"
+          render={props => <SubNav {...props} data={this.state.data} />}
+        />
+        <Route
+          path="/:itemName/:subItem"
```

Comment:
This is pretty advanced stuff for day 2 of React Router, being able to make a nested dynamic path.